saleor/product/models/base.py

```diff
@@ -71,11 +90,21 @@ def get_full_path(self, ancestors=None):
         return '/'.join([node.slug for node in nodes])
 
     def set_hidden_descendants(self, hidden):
+        # type: (bool) -> None
         self.get_descendants().update(hidden=hidden)
 
 
 @python_2_unicode_compatible
 class ProductClass(models.Model):
+    """representation of Product class e.g. t-shirt
```

Comment:
r->R
These docstrings explains nothining more than name of this class. How about: 
"Class describes common part of multiple products and its variant attributes"